core/cas-server-core-tickets-api/src/main/java/org/apereo/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -58,14 +59,27 @@ public void addTicket(final Ticket ticket) throws Exception {
 
     @Override
     public Ticket getTicket(final String ticketId) {
-        return getTicket(ticketId, ticket -> {
+        Ticket returnTicket = getTicket(ticketId, ticket -> {
```

Comment:
I don't quite follow why you'd need an explicit type here; Would `val returnTicket` work?